packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -424,6 +424,11 @@ function persistLocalEnvConfig(context) {
 }
 
 function getCurrentConfig(context) {
+  const { envName } = context.amplify.getEnvInfo();
+  return getConfigForEnv(context, envName);
+}
+
+function getConfigForEnv(context, envName) {
```

Comment:
why are we splitting the getCurrentConfig method into two? while the envName argument is not used at all in the new method getConfigForEnv(.)